api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/PaymentCycleVoucherEntitlementTestDataFactory.java

```diff
@@ -42,6 +43,20 @@ public static PaymentCycleVoucherEntitlement aPaymentCycleVoucherEntitlement(Loc
         );
     }
 
+    public static PaymentCycleVoucherEntitlement aPaymentCycleVoucherEntitlementWithBackdatedVouchersForYoungestChild(
+            LocalDate startDate, List<LocalDate> childrensDob) {
+        LocalDate birthdate = childrensDob.stream().max(LocalDate::compareTo).get();
+        return PaymentCycleVoucherEntitlement.builder()
+                .backdatedVouchers((int) ChronoUnit.WEEKS.between(birthdate, startDate))
+                .voucherEntitlements(Arrays.asList(
+                        aVoucherEntitlement(startDate, childrensDob, null),
+                        aVoucherEntitlement(startDate.plusWeeks(1), childrensDob, null),
+                        aVoucherEntitlement(startDate.plusWeeks(2), childrensDob, null),
+                        aVoucherEntitlement(startDate.plusWeeks(3), childrensDob, null)
```

Comment:
Not a part of this PR, but as I missed the original then I thought it worth a comment. I think the `aVoucherEntitlement` method name should be improved as it is a little smarter than a normal TestDataFactory method. When we get the chance, it should probably be renamed to something like `aVoucherEntitlementBasedOnChildrensDob`.